azext_iot/central/params.py

```diff
@@ -654,3 +661,261 @@ def load_central_arguments(self, _):
+            " The request body must contain GroupSymmetricKeyAttestation or GroupX509Attestation.",
+        )
+        context.argument(
+            "display_name",
+            options_list=["--display-name"],
+            help="Display name of the enrollment group."
+        )
+        context.argument(
+            "type",
+            options_list=["--type"],
+            help="Type of devices that connect through the group."
+        )
+        context.argument(
+            "enabled",
+            options_list=["--enabled"],
+            arg_type=get_three_state_flag(),
+            help="Whether the devices using the group are allowed to connect to IoT Central. True or False."
+        )
+        context.argument(
+            "certificate",
+            options_list=["--certificate", "--cert"],
+            help="The string representation of this x509 certificate.",
+        )
+        context.argument(
+            "verified",
+            options_list=["--verified", "--v"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate has been verified. True or False.",
+        )
+
+    with self.argument_context("iot central enrollment-group update") as context:
+        context.argument(
+            "attestation",
+            options_list=["--attestation", "--at"],
+            help="The attestation mechanism for the enrollment group. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/attestation.json]"
+            " [Example of stringified JSON:[{<Attestation Data JSON>}]."
+            " The request body must contain GroupSymmetricKeyAttestation or GroupX509Attestation.",
+        )
+        context.argument(
+            "display_name",
+            options_list=["--display-name"],
+            help="Display name of the enrollment group."
+        )
+        context.argument(
+            "type",
+            options_list=["--type"],
+            help="Type of devices that connect through the group."
+        )
+        context.argument(
+            "enabled",
+            options_list=["--enabled"],
+            arg_type=get_three_state_flag(),
+            help="Whether the devices using the group are allowed to connect to IoT Central. True or False."
+        )
+        context.argument(
+            "certificate",
+            options_list=["--certificate", "--cert"],
+            help="The string representation of this x509 certificate.",
+        )
+        context.argument(
+            "verified",
+            options_list=["--verified", "--v"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate has been verified. True or False.",
+        )
+        context.argument(
+            "remove_x509",
+            options_list=["--remove-x509"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate should be removed from the group. True or False."
```

Comment:
what happens if you have only one cert and you remove it?

Reply:
Nothing happens? I don't think any enrollment groups require a cert.